client/web/src/cody/onboarding/instructions/NeoVim.tsx

```diff
@@ -36,7 +36,8 @@ export function NeoVimInstructions({
                                         Open the plugin repo on GitHub
                                     </Text>
                                     <Text className="text-muted mb-0" size="small">
-                                        Download the latest version of the plugin from the repo.
+                                        Follow the instructions detailed in the <strong>Readme.md</strong> file to
```

Comment:
I think it's lowercase?
```suggestion
                                        Follow the instructions detailed in the <strong>readme.md</strong> file to
```